source/install/config-mattermost-server.rst

```diff
@@ -0,0 +1,33 @@
+Configuring Mattermost Server
+=============================
+
+1. Navigate to ``https://mattermost.example.com`` and create a team and user.
+
+  The first user in the system is automatically granted the
+  ``system_admin`` role, which gives you access to the System Console.
+
+3. Open the System Console. Click your username at the top left of navigation panel. In the menu that opens, click **System Console**.
+
+4. Setup an SMTP email service. Click **Notification** > **Email** and make the following changes. The example below assumes AmazonSES.
+
+    a. Set **Send Email Notifications** to ``true``
+    b. Set **Require Email Verification** to ``true``
+    c. Set **Feedback Name** to ``No-Reply``
+    d. Set **Feedback Email** to ``mattermost@example.com``
+    e. Set **SMTP Username** to ``[YOUR_SMTP_USERNAME]``
+    f. Set **SMTP Password** to ``[YOUR_SMTP_PASSWORD]``
+    g. Set **SMTP Server** to ``email-smtp.us-east-1.amazonaws.com``
+    h. Set **SMTP Port** to ``465``
+    i. Set **Connection Security** to ``TLS``
+    j. Save the Settings
+
+5. Click **FILES** > **Storage** and change **Local Directory Location** from ``./data/`` to ``/mattermost/data``
+
+6. Update **General** > **Logging** settings:
+
+   -  Set **Log to The Console** to ``false``
+
+7. Feel free to modify other settings.
+8. Restart the Mattermost Service.
```

Comment:
I mostly agree with you, but I think we should leave it as is for now because configuring is outside the scope of the install project.

The configuring section requires, IMHO, a complete rework. For example, this page is really about configuring notifications... that's why you need to set up an SMTP server... but it's written as if you *need* to set up SMTP for some reason, and in a specific AWS region, with no real explanation why you would want to do that. Plus, there's an existing separate section for setting up SMTP: https://docs.mattermost.com/install/smtp-email-setup.html#

The System Console has lots of description about what the fields are and how to fill them out, so most of the info in this section is either superfluous or incomplete, depending on your point of view. Superfluous in the sense that it adds almost no additional information or instructions on setting up notifications beyond what is available in the System Console. Incomplete in the sense that it covers only 3 of the tasks that you can do in the System Console, and furthermore, gives no idea as to why you need to do them or why you should do them. For example, if you must change `./data` to `/mattermost/data`, then why is `/mattermost/data` not the default? Maybe you don't need to change this value?

At minimum the Mattermost config section needs to tell you how to get to the System Console, with an overview of what you can change and why, and a note about having to restart the server (or not... does every change require a server restart, or just the logging changes, or just the SMTP changes, or what?) Also, are some changes essential? How do you optimize for production? For security? These are questions that we should answer for users in this section.